Sources/AppCoinsSDK/Domain/Entities/Currency.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Currency.swift
+//
+//
+//  Created by Graciano Caldeira on 16/08/2024.
+//
+
+import Foundation
+
+internal class Currency: Codable {
+    
+    let currency: String
+    let label: String
+    let sign: String
+    let type: String
+    let flag: URL?
+    
+    init(currencyRaw: CurrencyRaw) {
+        self.currency = currencyRaw.currency
+        self.label = currencyRaw.label
+        self.sign = currencyRaw.sign
+        self.type = currencyRaw.type
+        if let url = URL(string: currencyRaw.flag) {
```

Comment:
`self.url = URL(string: currencyRaw.flag)`

Will do the trick, no? No need for 3 lines

Reply:
Done